codespell_lib/data/dictionary.txt

```diff
@@ -17575,6 +17588,8 @@ itslev->itself
 itteration->iteration
 itterations->iterations
 iunior->junior
+iverse->diverse, inverse,
+iversed->inverse
```

Comment:
This says this correction is also valid, but probably rarer:
https://www.dictionary.com/browse/inversed
```suggestion
iversed->inverse, inversed,
```